### PR TITLE
chore(repositories): new pages structure

### PR DESCRIPTION
**Pull request by olarclara**
- Renames 'Repositories' to 'Legacy repositories'
- Creates two new pages:
  - Open source
  - All repositories (v2)

qkhf8z-3000 preview csb app_dashboard_repositories_open-source_workspace=411542e0-c5fb-4ee8-85c9-65c8cfd18e52

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6890/commits/81a4b93488d4ccf1f5e5791bf013ddf1f15f75fe is at https://pr6890.build.csb.dev/s/new.

**Review by tristandubbeld — COMMENTED** [review]
Looks good to me! Only thing that happened to me is when clicking a folder on the legacy repos page, it redirected me to another page instead of showing the contents of the folder. Doe we need sub-routes for these folders?

File: packages/app/src/app/pages/Dashboard/Content/routes/LegacyRepositories/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  const param = params.path || '';
+  const home = !param || param === '/';
+  const actions = useActions();
+  const {
+    activeTeam,
+    dashboard: { sandboxes, viewMode },
+  } = useAppState();
+
+  React.useEffect(() => {
+    const p = home ? null : param;
```
[the remark this answers is not in the document]
Comment (olarclara) [reply packages/app/src/app/pages/Dashboard/Content/routes/LegacyRepositories/index.tsx:28]: @tristandubbeld yep! could use some renaming, I guess.

**Comment by olarclara** [comment]
@tristandubbeld good catch. fixed here: b615121a73d7732cf04565fd996a7bd51aa8e5d4

**Review by tristandubbeld — APPROVED** [review]
Looks good to me!

**Review by alexnm — APPROVED** [review]
💯

**Pull request merged**